client/src/store/addPost/action.ts

```diff
@@ -37,26 +37,26 @@ export const setDescriptionForPost = (description: string): { type: string, payl
     payload: description,
 });
 
-export const resetAddPost = (): { type: string, payload: any } => ({
+export const resetAddPost = (username: string): { type: string, payload: any, previousPage: void } => ({
     type: RESET_ADD_POST,
     payload: '',
+    previousPage: history.push(`profile/${username}`),
```

Comment:
previousPage: history.push(`profile/${username}`) = what for?

Reply:
fixed